plugins/eksporter.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: eksporter
+spec:
+  version: "v1.0.0"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/Kyrremann/kubeflow-eksporter/releases/download/v1.0.0/eksporter.tar.gz
+    sha256: "1c13fac59c1c306db8498f394ee4cf1ba98eaa44c6f404fc2fad505533f00835"
+    files:
+    - from: "/eksporter.rb"
+      to: "."
+    bin: "./eksporter.rb"
+  shortDescription: Eksport resources without generated fields
```

Comment:
I think you are giving a false promise here. From what I can tell, your plugin removes a pre-defined set of fields from the manifest. For example, `metadata.labels` and `metadata.namespace`. Both of those could have been set explicitly by the user. There is no way for you to differentiate between automatically set defaults or user-specified values in those cases.

So I think you need to revise your description :)

Reply:
I agree, and I'll borrow some text from what you wrote:

> Export resources and strippes away a pre-defined set of fields